packages/mdc-bottom-line/README.md

```diff
@@ -59,13 +59,14 @@ Method Signature | Description
 `setAttr(attr: string, value: string) => void` | Sets an attribute with a given value on the root element
 `registerEventHandler(evtType: string, handler: EventListener) => void` | Registers an event listener on the root element for a given event
 `deregisterEventHandler(handler: EventListener) => void` | Deregisters an event listener on the root element for a given event
-`notifyAnimationEnd() => void` | Emits a custom event "MDCTextFieldBottomLine:animation-end" denoting the bottom line has finished its animation; either the activate or deactivate animation
+`notifyAnimationEnd() => void` | Emits a custom event "MDCBottomLine:animation-end" denoting the bottom line has finished its animation; either the activate or deactivate animation
 
-### `MDCTextFieldBottomLineFoundation`
+### `MDCBottomLineFoundation`
 
 Method Signature | Description
 --- | ---
 `activate() => void` | Activates the bottom line
-`deactivate => void` | Deactivates the bottom line
+`deactivate() => void` | Immediately deactivates the bottom line
+`deactivateFocus() => void` | Sets `isActive_` flag to false, allowing the animation to finish before deactivating. 
```

Comment:
this description is an internal implementation detail. Be more general about what `deactivateFocus` is trying to accomplish. Do we still need `deactivate`? Shouldn't we always wait for the animation to finish before deactivating?

Reply:
We could remove the `deactivateFocus` field and combine it's functionality into `deactivate`